src/main/java/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobProperty.java

```diff
@@ -71,7 +71,7 @@ public PipelineTriggersJobProperty(List<Trigger> triggers) {
     }
 
     public void setTriggers(List<Trigger<?>> triggers) {
-        this.triggers = triggers;
+        this.triggers.addAll(triggers);
```

Comment:
This is rather surprising. I would not expect direct API calls, but they would work weirdly if so.

Would it not be simpler to replace the `Collections.unmodifiableList()`, wherever that is, with `new ArrayList<>()`?

Or to make this code be

```java
this.triggers = new ArrayList<>(triggers);
```